### PR TITLE
feat: added text utilities and lists

### PR DESCRIPTION
**Pull request by harshalini**
Added text utilities.
In text utilities there are headings, small text, gray text and center text.
Added lists.
In lists, there are spaced list (normal list) and stacked list.
Check this here ->
https://youthful-jennings-006193.netlify.app/documentation/documentation.html
Screenshot (223)
Screenshot (224)
Screenshot (225)
Screenshot (226)
Screenshot (227)

### CODE REVIEW

File: components/texts/text-list.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+}
+.medium-text {
+    font-size: 20px;
+}
+.gr-tx {
+    color: darkgray;
+}
+
+.center-text {
+    width: 500px;
```
Comment (munjalSaksham71) [inline components/texts/text-list.css]: can you change it to rems?

File: components/texts/text-list.css [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    margin: 0.5rem;
+}
+
+.notification-card {
+    display: flex;
+    padding: 0rem 4rem;
+    list-style: none;
+    justify-content: center;
+    align-items: center;
+    padding: 1rem 0rem 1rem 0rem;
```
Comment (munjalSaksham71) [inline components/texts/text-list.css:38]: use padding: 16px 0px;  Other two will automatically take from this two..

File: documentation/documentation.css [hunk 3]
```diff
@@ -47,19 +47,19 @@ ul {
     font-size: 1.5rem;
     background-color: var(--white);
     margin-left: 13rem;
-    top: 100px;
+    top: 80px;
```
Comment (munjalSaksham71) [inline documentation/documentation.css]: Use rems here too

**Review by munjalSaksham71 — COMMENTED** [review]
Added Some Minor Comments, and then you are good to merge

File: components/texts/text-list.css (hunk 2 above)
Comment (sks-21) [inline components/texts/text-list.css]: padding: 1rem 0rem ;
means 1rem for top and bottom and 0rem for both right and left. It is one of the shorthand formats

**Review by sks-21 — APPROVED** [review]
The code looks nice. Just one suggestion, you should prefer relative units over absolute units.

File: components/texts/text-list.css (hunk 2 above)
Comment (munjalSaksham71) [reply components/texts/text-list.css:38]: Yes Thats what i wanted to say exactly

File: components/texts/text-list.css (hunk 1 above)
Comment (harshalini) [reply components/texts/text-list.css]: Made the changes you suggested, please review it again

File: components/texts/text-list.css (hunk 2 above)
Comment (harshalini) [reply components/texts/text-list.css]: Yeah, thanks for suggestion

**Review by munjalSaksham71 — APPROVED** [review]

**Pull request merged**